documentation/src/main/wiki/_Sidebar.md

```diff
@@ -13,6 +13,7 @@
         * [[Suite Structure|SuiteStructure]]
     * [[Running Suite|RunningSuite]]
         * [[Tracking Progress|TrackingProgress]]
+    * [[Best practices for writing test scripts|BestPractices]]
```

Comment:
How about placing it under `Defining Suite` as `Best practices`?

Reply:
Will do.